tools/viewer/main.rs

```diff
@@ -28,7 +28,7 @@ struct Cli {
     #[structopt(name = "path to .60 file", parse(from_os_str))]
     path: std::path::PathBuf,
 
-    /// The style name ('native', or 'ugly')
+    /// The style name ('native', or 'ulgy')
```

Comment:
```suggestion
    /// The style name ('native', or 'ugly')
```

This was fixed in 21f9850b4378cd05e80455619701524d59575b72 :)

Reply:
i don't know why this was overwritten, maybe a did a pull and i had some unsaved changed in that file. vscode did not show me a warning